rustfmt.toml

```diff
@@ -4,6 +4,7 @@ use_small_heuristics = "Max"
 merge_derives = false
 group_imports = "StdExternalCrate"
 imports_granularity = "Module"
+use_field_init_shorthand = true
```

Comment:
This PR changes the fmt config, that's why it causes unrelated PRs to break.

Reply:
... Thanks, I missed that part.